helm/chart/router/templates/deployment.yaml

```diff
@@ -23,8 +23,9 @@ spec:
       {{- include "router.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- if or .Values.podAnnotations .Values.supergraphFile }}
       annotations:
+        kubectl.kubernetes.io/default-container: {{ .Chart.Name }}
+      {{- if or .Values.podAnnotations .Values.supergraphFile }}
```

Comment:
I think this if can be removed as the inner ifs contain the same logic.
```suggestion

```

Reply:
You are right. I'll need to remove line 35 as well.

Reply:
done: 70b1a230e9be2dd056d97a4c1b9c9c72d9dc8e06